pkg/v3/types/trigger.go

```diff
@@ -76,6 +74,7 @@ func (e LogTriggerExtension) LogIdentifier() []byte {
 	binary.BigEndian.PutUint32(indexBytes, e.Index)
 	return bytes.Join([][]byte{
 		e.TxHash[:],
+		e.BlockHash[:],
```

Comment:
Is there any significance to the order here? Does the block hash need to go before the index bytes or can it be added at any point?

Reply:
good point. this needs to match the dedupe ID on contract which is under discussion

Reply:
aligned with contract dedupe key order in https://github.com/smartcontractkit/chainlink/pull/10476